packages/next-templates/src/app/card-component/card/page.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { Heading } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import Image from 'next/image';
+import React, { FC, HTMLAttributes } from 'react';
+import './index.style.css';
+
+interface CardListItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'children'> {
+  headerlevel: number;
+  title?: string;
+  preheading?: string;
```

Comment:
`pre-heading` has a dash, so I would use `preHeading` as casing to be consistent.